src/utils/api.js

```diff
@@ -173,6 +173,13 @@ export const filterActionsByUser = (actions, usersArray) => {
   );
 };
 
+export const filterActionsByActionTypes = (actions, actionsArray) => {
+  const actionKey = 'actionType';
+  return actions.filter((action) =>
+    actionsArray.some((act) => act.value === action[actionKey]),
```

Comment:
```suggestion
actionsArray.some((act) => act.value === action.actionType)
```

Reply:
I followed the convention of the previous function, then should I also change the previous one?
![image](https://user-images.githubusercontent.com/44131581/185750220-59b9dffd-c14e-4bd3-b160-21e53479f499.png)